### DISCUSSION TITLE
Naming Convention not showing in hasura.

### DISCUSSION THREAD

**Discussion by ManivannanMurugavel** [hasura/graphql-engine#10027] [Question]
I added the HASURA_GRAPHQL_EXPERIMENTAL_FEATURES=naming_convention, but the "GraphQL Field Customization" section  not showing in page. I am used the hasura version is v2.8.4..

![image](https://github.com/hasura/graphql-engine/assets/20641562/42869267-22a2-43f5-97a5-803c184ac03a)

Can anyone help me?

**Comment by SamirTalwar** [comment]
Looks like this feature was released in v2.9.0 ([changelog](https://hasura.io/changelog/cloud/v2.9.0-cloud.1)).

In general, it's probably worth upgrading to the latest version.

**Comment by ManivannanMurugavel** [comment]
@SamirTalwar 
I am using the docker-compose deployment. 
If I change to ```image: hasura/graphql-engine:v2.35.1```, the data will be lost or not?

**Reply by SamirTalwar** [reply]
All your data is stored in your database, which Hasura does not manage. If you are storing your metadata in the same PostgreSQL database (this is the default behavior), then it should upgrade that metadata when you upgrade the version.

That said, I recommend taking a snapshot of it before upgrading using the Hasura CLI, by running `hasura metadata export` ([docs](https://hasura.io/docs/latest/hasura-cli/commands/hasura_metadata_export/)). If you keep the metadata in version control and synced with Hasura, you will easily be able to move it to and from different instances.

**Reply by ManivannanMurugavel** [reply]
Thanks for your updates..
